modules/gateways/blockonomics/pollTransactionStatus.php

```diff
@@ -151,13 +151,18 @@ function getPaymentAmount($bits, $tokenAmount, $order) {
 
 
 do {
-    sleep(60); // Wait for 60 seconds
-    $unconfirmedOrders = $blockonomics->getUnconfirmedUSDTOrders();
-    
-    if (!$unconfirmedOrders->isEmpty()) {
-        foreach ($unconfirmedOrders as $order) {
-            pollTransactionStatus($order);
-            sleep(1);
+    try {        
+        logMessage("Polling", "Starting to poll", "true");
+        sleep(1);
+        $unconfirmedOrders = $blockonomics->getUnconfirmedUSDTOrders();
+        
+        if (!$unconfirmedOrders->isEmpty()) {
+            foreach ($unconfirmedOrders as $order) {
+                pollTransactionStatus($order);
+                sleep(1);
```

Comment:
The `sleep(1)` here might not be necessary since we're already sleeping at the beginning of each main loop iteration.